querydsl-mongodb/src/main/java/com/querydsl/mongodb/morphia/MorphiaSerializer.java

```diff
@@ -36,9 +39,20 @@
 public class MorphiaSerializer extends MongodbSerializer {
 
     private final Morphia morphia;
+    private final GeometryQueryConverter geometryQueryConverter;
 
     public MorphiaSerializer(Morphia morphia) {
-        this.morphia = morphia;
+        this.morphia = morphia == null ? new Morphia() : morphia;
```

Comment:
this implies that null would be a valid argument to the constructor, which it shouldn't be